specification/openai/resource-manager/Microsoft.OpenAI/preview/2021-09-01-preview/openai.json

```diff
@@ -0,0 +1,2029 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "OpenAIManagementClient",
```

Comment:
I heard the naming convention is all abbreviation should only have first letter capitalized, so it sound like this need to be "OpenAi"? But "OpenAI" is a branding name, so can we keep the casing here as "OpenAI"?

Reply:
This case Microsoft is paying for the brand. IMO we should keep the brand name.